src/lib.rs

```diff
@@ -95,9 +95,9 @@
 #![deny(non_camel_case_types)]
 #![deny(non_snake_case)]
 #![deny(unused_mut)]
-#![deny(dead_code)]
-#![deny(unused_imports)]
-#![deny(missing_docs)]
+// #![deny(dead_code)]
+// #![deny(unused_imports)]
+// #![deny(missing_docs)]
```

Comment:
in 73dd23e247dc47fd4d6d50244bc30e27abd3b487, this is fine. 


Adding this change to check this is re-enabled again.